### PR TITLE
Adds `tk fmt` documentation

### LINKED ISSUE #407: Add `tk fmt` documentation
I can't find any mention of the `tk fmt` command on the Tanka docs website. A small mantion of it would help tremendously for newcomers to the tool 🙂

### PR DESCRIPTION
**Pull request by heuristicAL**
Fixes #407

### CODE REVIEW

**Review by Duologic — APPROVED** [review]
Look good to me, I don't know how the rendering of the webpage works tho.

File: docs/docs/formatting.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+name: "Formatting"
+route: "/formatting"
+menu: "References"
+---
+
+# File Formatting
+
+Tanka supports formatting for all `jsonnet` and `libsonnet` files using the `tk fmt` command.
+
+By default, the command excludes all `vendor` directories and any files in the immediate directory.
```
Comment (sh0rez) [inline docs/docs/formatting.md:11]: > and any files in the immediate directory

Could you explain what you mean by this sentence? To me it reads like `tk fmt .` would not format the files in the current directory, which is not true afaict

File: docs/docs/formatting.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+```bash
+# Run for current and child directories. Run this in the root of the project to format all your files.
+$ tk fmt .
```
Comment (sh0rez) [inline docs/docs/formatting.md:15]: I'd love to see some more examples here:

1. Formatting single files (`tk fmt myFile.jsonnet`)
2. Testing instead if formatted (`tk fmt -t myFile.jsonnet`)
3. Verbose mode (`tk fmt -v`)

**Review by sh0rez — CHANGES_REQUESTED** [review]
Looking nice!

File: docs/docs/formatting.md (hunk 1 above)
Comment (heuristicAL) [reply docs/docs/formatting.md:11]: You're quite right! Let me reword this.

**Review by sh0rez — APPROVED** [review]
Nice work! Congrats on your first PR :rocket:

**Pull request merged**